modules/core/src/main/java/org/apache/ignite/internal/processors/cache/KeyCacheObjectImpl.java

```diff
@@ -34,7 +34,7 @@ public class KeyCacheObjectImpl extends CacheObjectAdapter implements KeyCacheOb
     private static final long serialVersionUID = 0L;
 
     /** */
-    private int part = -1;
```

Comment:
Let's refactor only IgniteUuid in Messages in this patch.

Reply:
rollbacked